plugins/elasticsearch/src/connection.ts

```
@@ -0,0 +1,338 @@
+        if (!res.ok) {
+          throw new Error(
+            `Elasticsearch cluster-info request failed: HTTP ${res.status} ${res.statusText}`,
+          );
+        }
+
+        return normalizeClusterInfo(await res.json());
+      } catch (err) {
+        if (controller.signal.aborted && !closed) {
+          throw new Error(
+            `Elasticsearch cluster-info request timed out after ${timeoutMs}ms`,
+          );
```

ESLint

[error] There is no `cause` attached to the symptom error being thrown.

(preserve-caught-error)